README.md

```diff
@@ -32,147 +35,105 @@ a new line, e.g.:
+An example of fetching data is shown below and in the [notebook](examples/example_artscraper.ipynb). 
 
-To download data from GoogleArt it is necessary to install 
-[Firefox](https://www.mozilla.org/en-US/firefox/new/) and `geckodriver`. 
-
-There are two options to download the geckodriver.
-- Using a package manager (recommended on Linux/OS X): On Linux, geckodriver 
-is most likely available through the package manager for your distribution
-(e.g. on Ubuntu `sudo apt install firefox-geckodriver`. On OS X, it is the 
-easiest to install it through either [brew](https://formulae.brew.sh/formula/geckodriver#default) or [macports](https://ports.macports.org/port/geckodriver/) (e.g. `brew install geckodriver`). Depending on your settings, you might need to add the directory where the geckodriver resides to the PATH variable. 
-- Downloading it [from here](https://github.com/mozilla/geckodriver/releases) and making it available to the code. For example in Windows you can download the file `geckodriver-v0.31.0-win64.zip`, place the driver in the directory of your code, and specify the path when you initialize the GoogleArtScraper. For example:
 ```python
-with GoogleArtScraper(geckodriver_path="./geckodriver.exe") as scraper:
-    ...
-```
-
-Make sure that you have a recent version of geckodriver, because selenium (a non-python dependency used in the GoogleArt scraper) uses features that were only recently introduced 
-in geckodriver. We have only tested the scraping on Linux/Firefox and OSX/Firefox.
-
 
-## Download images and metadata (interactive)
+from artscraper import WikiArtScraper
 
-An example of fetching data is shown in an
-[example](examples/example_artscraper.ipynb) notebook. 
+art_url = "https://www.wikiart.org/en/edvard-munch/anxiety-1894"
 
-Assuming the WikiArtScraper
-is used, we can download the data from a link with:
+with WikiArtScraper(output_dir="data") as scraper:
+    scraper.load_link(art_url)
+    scraper.save_metadata() 
+    scraper.save_image()
 
-```python
+```
 
-from artscraper import WikiArtScraper
+This will store both the image itself and the metadata in separate folders. If
+you use ArtScraper in this way, it will skip images/metadata that is already
+present. Remove the directory to force it to redownload it. 
 
-scraper = WikiArtScraper()
+Results:
 
-# Load the URL into the scraper.
-scraper.load_link("some URL")
+[<img src="https://uploads5.wikiart.org/images/edvard-munch/anxiety-1894.jpg" weight="20">](https://www.wikiart.org/en/edvard-munch/anxiety-1894)
 
-# Get the metadata
-metadata = scraper.get_metadata()
 
-# Save the image to some local file
-scraper.save_image("wiki.jpg")
+## 3. Downloading art from Google Arts & Culture
 
-# Release resources
-scraper.close()
-```
+To download data from GoogleArt it is necessary to install 
+[Firefox](https://www.mozilla.org/en-US/firefox/new/) and `geckodriver`. Geckodriver is installed automatically when you run the code for the first time.
```

Comment:
Is it not confusing to first say that geckodriver needs to be installed, and then say that it is installed automatically? Maybe it is better to leave out any mention of geckodriver?